components/style/logical_geometry.rs

```diff
@@ -123,11 +123,6 @@ impl WritingMode {
         !self.intersects(WritingMode::RTL)
     }
 
-    #[inline]
-    pub fn is_sideways(&self) -> bool {
```

Comment:
This is probably still useful, even if servo doesn't support sideways writing modes yet.